packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -186,7 +186,7 @@ hub.post("/", async (req, res) => {
               spokeName: spokeName,
               errorReported: errorOutput.errorOutputs[spokeName].execResponse
                 ? errorOutput.errorOutputs[spokeName].execResponse.stderr
-                : JSON.stringify(errorOutput.errorOutputs[spokeName])
+                : errorOutput.errorOutputs[spokeName]
```

Comment:
this enhances logs a bit. before:
![image](https://user-images.githubusercontent.com/12886084/107240144-409ef380-6a32-11eb-9240-53ba3e67da12.png)


after:
![image](https://user-images.githubusercontent.com/12886084/107240170-485e9800-6a32-11eb-8075-e897bea62987.png)